frontend/src/main.js

```diff
@@ -47,6 +48,7 @@ if (version === false) {
       { default: axios },
       { default: Snotify },
       { default: VueCookie },
+      ShortKey,
```

Comment:
funktioniert das? Wir wollten es doch local als directive verwenden, oder?

Reply:
does not work